terraform/modules/hub/files/cloud-init/prometheus.sh

```diff
@@ -131,10 +131,40 @@ Environment=HTTP_PROXY=${egress_proxy_url_with_protocol}
 Environment=HTTPS_PROXY=${egress_proxy_url_with_protocol}
 EOF
 
-cat <<EOF > /etc/prometheus/prometheus.yml
-${prometheus_config}
-EOF
-
 systemctl daemon-reload
 systemctl enable  prometheus
 systemctl restart prometheus
+
+cat <<EOF > /usr/bin/cronitor-prometheus-config-update.sh
+#!/usr/bin/env bash
+set -ueo pipefail
+
+: "${CRONITOR_URL:?CRONITOR_URL not set}"
```

Comment:
```suggestion
: "$${CRONITOR_URL:?CRONITOR_URL not set}"
```

Reply:
Terraform is attempting to interpolate this bashy boi

`$$` escapes the `$`

Reply:
👀